BLUESPAWN-win-client/headers/util/permissions/permissions.h

```diff
@@ -212,6 +274,70 @@ namespace Permissions{
 		 * @return The name of the owner
 		 */
 		virtual std::wstring ToString() const;
+
+		/**
+		* Function to get a list of privileges the owner has
+		*
+		* @return the list of privileges an owner explicity has, and if they 
+		*     are a user, includes the privileges granted to them by the
+		*     groups they're in.
+		*/
+		std::vector<std::wstring> GetPrivileges();
+
+		/**
+		* Function to check if the owner has a certain privilege
+		*
+		* @param wPriv the name of the privilege to check for. Names should
+		*    be given as constants listed here: https://docs.microsoft.com/en-us/windows/win32/secauthz/privilege-constants
+		* 
+		* @return true if the owner has the privilege, false otherwise
+		*/
+		bool HasPrivilege(IN const std::wstring& wPriv);
```

Comment:
it might be a good idea to eventually make something better to represent a privilege. Strings don't feel like a natural program representation. This is fine for now though